www_src/pages/section-2/section-2.jsx

```diff
@@ -16,24 +25,35 @@ var App = React.createClass({
     var elWrapper = this.refs.wrapper.getDOMNode();
     this.refs.masterGrid.setContainerDimensions(elWrapper.clientWidth, elWrapper.clientHeight);
 
-    this.refs.masterGrid.showOverview();
+    setTimeout(function() {
+      this.refs.masterGrid.setZoomLevel(this.state.zoomLevel);
+    }.bind(this), 100);
   },
-  render: function () {
-
-    // TEMP: Change div w. zIndex to Draggable to enable drag
+  onZoomChange: function (event) {
+    console.log('ozc', event);
 
+    this.setState({
+      zoomLevel: event.pagesWide
+    });
+  },
+  render: function () {
     return (
       <div className="section-2">
         <div ref="wrapper" className="wrapper">
           <div zIndex={100}>
             <div>
-              <Grid ref="masterGrid" aspectRatio={"35:40"}/>
+              <Grid ref="masterGrid" aspectRatio={"35:40"} onZoomChange={ this.onZoomChange } />
             </div>
           </div>
         </div>
         <div className="segmented-control">
-          <button onClick={ this.showOverview }>-</button>
-          <button className="disabled">+</button>
+          <Hammer
+            className={ this.state.zoomLevel < 6.25 ? 'enabled' : 'disabled' }
```

Comment:
Should we store this zoom level in single place? Seems a little odd to have it embedded in the JSX twice like this.